packages/expo-updates/ios/EXUpdates/Logging/UpdatesErrorCode.swift

```diff
@@ -7,36 +7,36 @@ import Foundation
  */
 @objc(EXUpdatesErrorCode)
 public enum UpdatesErrorCode: Int {
-  case None = 0
-  case NoUpdatesAvailable = 1
-  case UpdateAssetsNotAvailable = 2
-  case UpdateServerUnreachable = 3
-  case UpdateHasInvalidSignature = 4
-  case UpdateFailedToLoad = 5
-  case AssetsFailedToLoad = 6
-  case JSRuntimeError = 7
+  case none = 0
+  case noUpdatesAvailable = 1
+  case updateAssetsNotAvailable = 2
+  case updateServerUnreachable = 3
+  case updateHasInvalidSignature = 4
+  case updateFailedToLoad = 5
+  case assetsFailedToLoad = 6
+  case jSRuntimeError = 7
```

Comment:
It looks a bit odd to write `jS` 😅 
```suggestion
  case jsRuntimeError = 7
```